src/lightning_app/README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
 
-<img src="https://pl-flash-data.s3.amazonaws.com/brandmark.png" width="400px">
+<img src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/brandmark.png" width="400px">
```

Comment:
why it is saved on Flash bucket?

Reply:
This is the buckets we were using for lightning-app. We could have created another one, but there were some weird permission errors.